sdks/python/apache_beam/testing/load_tests/load_test_metrics_utils.py

```diff
@@ -111,11 +112,15 @@ def _prepare_all_metrics(self, metrics):
       insert_rows.append(counter_dict)
 
     dists = metrics['distributions']
-    if len(dists) > 0:
-      runtime = RuntimeMetric(dists, submit_timestamp, metric_id)\
-        .as_dict()
-      insert_rows.append(runtime)
-
+    sorted_dists = sorted(dists, key=lambda x: x.key.metric.name)
+    for key, dists in groupby(sorted_dists, lambda x: x.key.metric.name):
+      dists = list(dists)
+      if key == RUNTIME_METRIC:
```

Comment:
It will be more robust to check group by metric namespace+name, not just name.

Reply:
Sure. Makes sense. I had to divide list into 2 (with our namespace and "other" namespaces).